builtin/logical/nomad/path_config_access.go

```diff
@@ -23,6 +27,12 @@ func pathConfigAccess(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: "Token for API calls",
 			},
+
+			"max_token_name_length": &framework.FieldSchema{
+				Type:        framework.TypeInt,
+				Description: "Max length for name of generated Nomad tokens",
+				Default:     maxTokenNameLength,
```

Comment:
One more thing here.  You should probably not set the default value in the path config.  This will mean we will store this value in storage and if we change the default again, they will have an explicit value.  You can just omit the Default.

Reply:
Omitting the default will leave a zero value in the struct and be serialized like that. We could change the `accessConfig.MaxTokenNameLength` to be an `*int` and use `nil` there, or just ignore the value when it's zero. I'll assume the later for now

Reply:
Or set it the default to -1 maybe?

Reply:
I think the zero value is fine, it is the same as default.  I don't think we need to do any int pointer work. Does this api support create vs update?  If so, you should be able to handle the case where they don't provide the value and just patch the updated fields.